kaggle_environments/envs/halite/halite.py

```diff
@@ -86,6 +86,11 @@ def populate_board(state, env):
     size = env.configuration.size
     uid_counter = 0
 
+    # Set seed for random number generators
+    if config.randomSeed != 0:
```

Comment:
I don't remember the behavior off the top of my head, but if you omit a default for randomSeed in the configuration will it get initialized to `None` here? I feel like it'd be more explicit to use `None` as our sentinel value than `0` if possible.

Reply:
Makes sense. The reason I didn't use `None` is because I don't know now to specify that value in JSON. See https://github.com/Kaggle/kaggle-environments/blob/8efa322d9c99a2b672a2a3c825ab6acf1b5819fa/kaggle_environments/envs/halite/halite.json#L59

Should I use `null`?

Reply:
TBH I don't remember off the top of my head, I'd give null a try and if that doesn't work try just omitting the default field completely.

Reply:
If neither of those work, lmk and we can look into adding a hook for specifying None in the schema